CMakeLists.txt

```diff
@@ -65,7 +68,7 @@ endif(BUILD_PYTHON_INTERFACE)
 set(ODRI_CONTROL_INTERFACE_SRC src/joint_modules.cpp src/imu.cpp src/robot.cpp
                                src/calibration.cpp src/utils.cpp)
 add_library(${PROJECT_NAME} SHARED ${ODRI_CONTROL_INTERFACE_SRC})
-target_link_libraries(${PROJECT_NAME} yaml-cpp)
+target_link_libraries(${PROJECT_NAME} yaml-cpp::yaml-cpp)
```

Comment:
not sure this would work with all yaml-cpp versions. Did you change this because you had a compilation without ?

Reply:
Looking at the git blame on yaml-cpp's repo, it seems like the exported target is only namespaced since last year:

https://github.com/jbeder/yaml-cpp/blame/master/CMakeLists.txt#L177

So indeed it wouldn't work with prior versions, so I also have doubts about this change

Reply:
Thanks. Current installed version on my system has `add_library(yaml-cpp::yaml-cpp SHARED IMPORTED)` and no `yaml-cpp` target or alias. so let us add a guard